src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEM.hpp

```diff
@@ -203,63 +203,63 @@ namespace extrinsicMeshData
 
 EXTRINSIC_MESH_DATA_TRAIT( Pressure_nm1,
                            "pressure_nm1",
-                           array1d< real64 >,
+                           real32_array,
```

Comment:
Question: why `real32_array` and not `array1d< real32 >`?

Reply:
There is no particular reason, they are aliases. I could have used either, do you want me to change it ?

Reply:
Yes please, we tend not to use those aliases too much.